docs/ReleaseNotes.md

```diff
@@ -136,6 +200,8 @@ interpolation.
 
 - Fixed error where calling undo resulted in an error
 
+***
```

Comment:
Maybe for 2020.10 we'll split the release notes into sub-pages so this becomes more manageable :D

Reply:
yeah, that way we can continue to be liberal with our use of gifs